manage.py

```diff
@@ -37,6 +37,15 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def fix_digit_identifier():
+    events = Event.query.all()
+    for event in events:
+        if event.identifier.isdigit():
+            event.identifier = event.identifier + 'ref'
```

Comment:
I have already done this as a workaround, then what is the need of this modification? This was a workaround, not a fix